### PR TITLE
chore: update main comment with improvements from magazine comment

### PR DESCRIPTION
**Pull request by nathggns**
Main comment had a couple of changes that were made after magazine comment was created, and magazine comment improved a couple of things that can and should be copied back to main comment. That's what this PR does.

### CODE REVIEW

**Comment by aronshamash** [comment]
Looks fine but I also noticed that in `article` storybook we only have `mainstandard` and `maincomment`

**Comment by nathggns** [comment]
> Looks fine but I also noticed that in `article` storybook we only have `mainstandard` and `maincomment`

@aronshamash I'll update this now

File: packages/article-main-comment/src/article-prop-types/article-prop-types.base.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const articlePagePropTypes = {
     })
   }),
   isLoading: PropTypes.bool,
-  receiveChildList: PropTypes.func.isRequired
```
Comment (colincclark) [inline packages/article-main-comment/src/article-prop-types/article-prop-types.base.js]: articles should always be wrapped in the track scroll depth HOC

Comment (nathggns) [reply packages/article-main-comment/src/article-prop-types/article-prop-types.base.js]: Oops… think I was over eager in removing things to remove warnings. Will fix for both templates now!

**Pull request merged**